docs/src/pages/components/text-fields/ValidationTextFields.js

```diff
@@ -30,7 +30,7 @@ export default function ValidationTextFields() {
         />
         <TextField
           error
-          id="standard-error"
+          id="standard-error-helper-text"
```

Comment:
Otherwise we got duplicate ids. Including this here so that we recover lighthouse for this page completely (except contrast).